x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_style_property.js

```diff
@@ -315,7 +306,7 @@ export class DynamicStyleProperty extends AbstractStyleProperty {
     return null;
   }
 
-  _renderCategoricalLegend({ isPointsOnly, isLinesOnly, symbolId }) {
+  _renderBreakedLegend({ isPointsOnly, isLinesOnly, symbolId }) {
     return (
       <CategoricalLegend
```

Comment:
Maybe `CategoricalLegend` needs to be renamed since it is now used for original scales as well.

Reply:
+1 renaming it to breaked-legend, as that maps to the other method names